index.fr.md

```diff
@@ -0,0 +1,226 @@
+
+Le type d'évaluation que vous pratiquez dépend de facteurs tels que l'étape à laquelle vous vous trouvez dans votre projet, par exemple, si vous en êtes au début de la recherche d'idées de conception, au test de différentes parties de vos prototypes, ou à la vérification finale de vos créations.
+
+**Mener des évaluations informelles tout au long du développement est plus efficace qu'effectuer un test d'utilisation formel en fin de projet.**
+
+## Les fondamentaux {#basics}
+
+Dans la plupart des cas, inclure les utilisateurs lors de l'évaluation suppose d' :
+
+-   inclure quelques utilisateurs en situation de handicap - et selon votre public cible, des utilisateurs âgés
+-   les inclure tout au long du développement pour effectuer des exemples de tâches sur vos prototypes pour vous permettre d'identifier quels aspects de la conception et du code peuvent être améliorés
+-   échanger à propos des problèmes d'accessibilité avec eux
+
+**Allez sur [[Inclure les utilisateurs dans les projets Web pour une accessibilité plus optimale et plus facile]](/planning/involving-users/)** pour trouver de l'aide dans [Inclure différents utilisateurs](/WAI/users/involving#diverse) et [expérience des utilisateurs interagissant avec le Web](/WAI/users/involving#experience)
+
+Comme pour toute évaluation avec des utilisateurs, inclure des utilisateurs de niveau basique, intermédiaire ou avancé, dépendra de vos utilisateurs cibles. Par exemple, si vous développez une application Web pour des comptables au sein d'une entreprise, vous opterez certainement pour des utilisateurs de technologies d'assistance d'un niveau avancé. Par contre, pour un site Web destiné aux demandes de prestation d'invalidité, vous travaillerez plutôt avec des utilisateurs de technologies d'assistance novices.
+
+<mark><strong>Attention :</strong> prêtez attention à toutes les données. <strong>Évitez de supposer que les données d'une personne présentant une certaine forme de handicap s'appliquent à toutes les personnes handicapées. </strong></mark>
+
+Une personne handicapée ne sait pas forcément comment d'autres personnes présentant le même type de handicap interagissent avec le Web. Elle pourrait ne pas en savoir assez sur les autres handicaps pour fournir une aide valable pour d'autres questions d'accessibilité. Il est préférable de rassembler des données de plusieurs types d'utilisateurs.
+
+Note : en plus de détecter des problèmes d'accessibilité, l'évaluation menée avec des utilisateurs handicapés révèlent des problèmes généraux d'utilisation qui touchent tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+
+## Analyse des problèmes d'accessibilité {#analyz}
+
+L'accessibilité Web dépend de l'association [des différents composants du développement Web avec l'interaction](/fundamentals/components/), dont les navigateurs Web, les technologies d'assistance, et le contenu Web.
```

Comment:
Maybe we can make the sentence more easy to grasp ? (inspired by the suggested translation of the destination page by @code-elegant)
"L'accessibilité Web nécessite que des composantes multiples du développement web et de l’interaction utilisateur s’articulent convenablement, [...]"